repose-aggregator/core/repose-core-api/src/test/scala/org/openrepose/core/filter/AbstractConfiguredFilterTest.scala

```diff
@@ -143,6 +167,18 @@ class AbstractConfiguredFilterTest
 
       filter.getInitialized shouldBe true
     }
+
+    it("should call doConfigurationUpdated") {
```

Comment:
Hm... as in being called after running the abstract filter code for the related method? I can certainly add a test for that, but do we actually care? I suppose we might if you want to be able to, say unsubscribe immediately from the configuration that the abstract filter subscribed to. Although in that case, I think a user would probably be better off just overriding the primary (non-extension) method.

I feel like I'm missing something here...